### PR TITLE
Entity clusters

### PR DESCRIPTION
**Pull request by gronders**
- add full bitcoin clusters tables (will add incremental later) 
- remove the weekly dbt_process from the old python method

### CODE REVIEW

File: models/gold/core/core__dim_entity_clusters.yml [hunk 1]
```diff
@@ -10,11 +10,9 @@ models:
         tests:
           - unique
           - not_null
-      - name: address_group
+      - name: group_id
```
Comment (eric-laurello) [inline models/gold/core/core__dim_entity_clusters.yml:13]: keep this as address_group

File: models/silver/labels/silver__clusters_changelog.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import datetime
+
+
+def model(dbt, session):
+
+    dbt.config(
+        materialized="table",
+        packages=["networkx"],
+        unique_key="ADDRESS",
+        tags=['entity_cluster']
```
Comment (forgxyz) [inline models/silver/labels/silver__clusters_changelog.py:13]: Add `enabled=False` for now, just no need to have it run an empty model tbh

File: models/silver/labels/silver__full_entity_cluster.sql [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+) }}
+
+SELECT
+    address,
+    group_id as address_group,
+    project_name,
+    _inserted_timestamp
+FROM
+    {{ source("bitcoin_bronze", "entity_clusters") }}
+
```
Comment (forgxyz) [inline models/silver/labels/silver__full_entity_cluster.sql:16]: Add incremental filter, here. Or if this is just always only going to select from the full cluster maybe a view is better? Currently, this is going to rebuild the table from the bronze source on every run, without the underlying data changing

File: models/sources.yml [hunk 4]
```diff
@@ -23,3 +23,9 @@ sources:
     schema: silver
     tables:
       - name: number_sequence
+
+  - name: bitcoin_bronze
+    database: bitcoin
+    schema: bronze
+    tables:
+      - name: entity_clusters
```
Comment (forgxyz) [inline models/sources.yml:31]: Newline

**Review by forgxyz — CHANGES_REQUESTED** [review]

File: models/sources.yml (hunk 4 above)
Comment (gronders) [reply models/sources.yml:31]: newline after?

**Review by forgxyz — APPROVED** [review]

**Pull request merged**